cmd/export/export.go

```diff
@@ -22,16 +23,48 @@ type ExportOptions struct {
 	ExportDir string
 	Context   string
 	Namespace string
+
+	//User Impersonation Flags
+	User   string
+	Group  []string
+	Extra  string
+	extras map[string][]string
+
 	genericclioptions.IOStreams
 }
 
+func (o *ExportOptions) setExtras() error {
+	keysAndStrings := strings.Split(o.Extra, ":")
+	if len(keysAndStrings)%2 != 0 {
```

Comment:
Why must the number of k/v pairs be even?
"key=string1,string2:key2=string3" from the example below is valid, but "key=string1,string2" and key=string1,string2:key2=string3:key3=string4" are invalid.

Reply:
Also, we've already made `;` the delimiter for optional args k/v pairs for the transform command, parsed similarly. Shouldn't we use `;` here for consistency?